services/applications/slicer/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM no-vnc-base:latest
```

Comment:
I think this was working for me, I think this was a local container. Can you confirm this change is needed?

Reply:
It was not working for me, so I changed